pkg/hubble/observer/local_observer.go

```diff
@@ -78,21 +77,21 @@ type LocalObserverServer struct {
 	log *logrus.Entry
 
 	// channel to receive events from observer server.
-	eventschan chan *observer.GetFlowsResponse
+	eventschan chan *observerpb.GetFlowsResponse
```

Comment:
nit: this field can actually be removed. it is not used anymore in embedded Hubble.

Reply:
Sure but I don't think it should be part of this PR. This is just code re-org/renaming without any other changes.